src/react-apps/applications/runtime/src/shared/resources/attachments/index.d.ts

```diff
@@ -9,20 +9,24 @@ export interface IAttachments {
   [attachmentType: string]: IAttachment[];
 }
 
-export interface IAttachmentListApiResponse {
-  type: string;
-  attachments: IAttachmentApiResponse[];
-}
-
-export interface IAttachmentApiResponse {
-  name: string;
-  size: number;
-  id: string;
-}
-
 export interface IAltinnWindow extends Window {
   org: string;
   app: string;
   instanceId: string;
   reportee: string;
 }
+
+export interface IDataElement {
```

Comment:
We already have an interface, IData. Remove this

Reply:
removed